docs/Configuration.md

```diff
@@ -237,6 +237,41 @@ Jest will fail if:
 * The `./src/api/very-important-module.js` file has less than 100% coverage.
 * Every remaining file combined has less than 50% coverage (`global`).
 
+### `forceCollectCoverage` [string]
+
+Default: `''`
+
+The pattern to enforceCollecting coverage even if the file is in the ignored
+normally.
+
+For example, if you have tests in source files named with `.t.js` extension as
+following:
+
+```javascript
+// sum.t.js
+
+export function sum(a,b) {
+  return a + b;
+}
+
+if (process.env.NODE_ENV === 'test') {
+  test('sum', () => {
+    expect(sum(1, 2)).toBe(3);
+  });
+}
+```
+
+You can collect coverage from those files with setting `forceCollectCoverage`.
+```json
+{
+  ...
+  "jest": {
+    "forceCollectCoverage": ".*\\.t.js$"
```

Comment:
Should it include `pattern` in the name to indicate it's a regexp?

Reply:
Seems like it's called `forceCoverageRegex` elsewhere, can you make sure it's consistent?

Reply:
I actually wrote it wrong on the docs :) it is `forceCoverageRegex`. Just updated the PR